src/script/api/script_road.cpp

```diff
@@ -136,6 +136,32 @@
 
 /* Helper functions for ScriptRoad::CanBuildConnectedRoadParts(). */
 
+/**
+ * Enumeration or the orientations of road parts.
+ *
+ * Technically DiagDirection could be used, but that allows simple conversions to/from integer. In this case that is not wanted.
```

Comment:
enum class DiagDirection? :D

Reply:
That'd be a massive change I do not really fancy doing as it probably needs to be a massive single commit.